### DISCUSSION TITLE
Cant find how to Merge  storages

### DISCUSSION THREAD

**Discussion by KalaChin** [IceWhaleTech/CasaOS#815] [General]
i can't find where the buttom is to merge storages,

**Comment by raller1028** [comment]
![image](https://user-images.githubusercontent.com/57336867/211967627-577d0779-c9e9-481b-a6fa-186a942d4622.png)

**Reply by KalaChin** [reply]
my web haven't shown this option , Is it because Beta version ？

**Reply by raller1028** [reply]
Is your casaos version 0.4.1-alpha?

**Reply by KalaChin** [reply]
version 0.4.0    <_> , it's not support ?

**Reply by raller1028** [reply]
yes, you can update to v0.4.1-alpha to merge storages>_<

**Reply by KalaChin** [reply]
okey，thank you

**Comment by xuguang2018** [comment]
怎么样算合并成功
